src/Features/CSharp/Portable/Completion/KeywordRecommenders/RefKeywordRecommender.cs

```diff
@@ -63,8 +63,7 @@ protected override bool IsValidContext(int position, CSharpSyntaxContext context
                 syntaxTree.IsPossibleLambdaParameterModifierContext(position, context.LeftToken, cancellationToken) ||
                 context.TargetToken.IsConstructorOrMethodParameterArgumentContext() ||
                 context.TargetToken.IsXmlCrefParameterModifierContext() ||
-                (syntaxTree.Options.Features.ContainsKey("refLocalsAndReturns") && 
```

Comment:
I am surprised this is still in there. I thought this was removed as a part of the work to make parsing not be conditional on features (only errors are conditional).